Assignment2/basicOp.js

```diff
@@ -0,0 +1,5 @@
+function basicOp(operation, value1, value2)
+{
+  // Code
+  return operation === '+' ? value1 + value2 : operation === '-' ? value1-value2 : operation === '*' ? value1*value2 : operation === '/' ? value1/value2 : 0

```

Comment:
when there are 4 cases, you can use "switch" here

Reply:
ok I will make this change